features/admin.tenants.v1/package.json

```diff
@@ -46,8 +47,6 @@
         "i18next-browser-languagedetector": "^6.1.5",
         "i18next-xhr-backend": "^3.2.2",
         "js-beautify": "^1.13.0",
-        "lodash-es": "^4.17.21",
```

Comment:
hope removing lodash-es is intentional.

Reply:
hey @pavinduLakshan this is because of using pnpm add dayjs command
order of few dependencies changed but not removed in this file

Reply:
Seems lodash-es has been removed for some reason. Couldn't see it has been added by looking at the diff. Could you please double check?

Reply:
my bad it was a mistake fixing it!

Reply:
fixed this!